src/Altinn.Apps/AppFrontend/react/altinn-app-frontend/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "altinn-app-frontend",
-  "version": "2.5.0",
+  "version": "2.5.1",
```

Comment:
Technically we should bump the minor version? 🤓

Reply:
Yup :)